rtc-video-quality/encoder_commands.py

```diff
@@ -127,7 +127,34 @@ def svt_command(job, temp_dir):
             '--preset', SVT_SPEED,
         ]
 
-    command = [binary_vars.SVT_ENC_BIN] + codec_params + common_params
+    elif encoder == 'svt-offline':
+
+        (fd, statfile) = tempfile.mkstemp(dir=temp_dir, suffix='.stat')
```

Comment:
I didn't see this in the python styleguide but I was wondering, is this required by the styleguide? The only reason I'm asking is because in the other parts of the code, parantheses were used for variable declaration. So I just wanted to keep it consistent with the other parts of the code. 

yapf formatter wasn't able to catch this

Reply:
https://google.github.io/styleguide/pyguide.html#33-parentheses says "Use parentheses sparingly."

That said, it's also good to be consistent with local style, so if the local style is to use parens there, it's fine to leave them.